desktop-widgets/tab-widgets/maintab.cpp

```diff
@@ -172,3 +178,15 @@ void MainTab::colorsChanged()
 	for (TabBase *widget: extraWidgets)
 		widget->updateUi(colorText);
 }
+
+// Called when dives changed. Checks whether the currently displayed
+// dive is affected by the change.
+bool MainTab::currentDiveAffected(const QVector<dive *> &dives) const
```

Comment:
something about this method name feels odd to me. Wouldn't `bool MainTab::includesCurrentDive(const QVector<dive *> &dives) const` be much more logical and intuitive?

Reply:
renamed.